src/main/java/org/broadinstitute/hellbender/tools/sv/Pedigree.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import java.io.*;
+import java.util.*;
+
+public class Pedigree {
```

Comment:
Naming this pedigree implies to me that it's a single pedigree object of some sort.  I would probably go with PedigreeFile, PedigreeTrio, or PedigreeCollection or something along those lines.   

However, we already have 2 different pedigree reader implementations.  In GATK there is PedReader which only supports reading but could be extended to support writing.  In picard there is PedFile which does both.  Is there a reason you can't use one of those?

Reply:
Another thing to look at, the picard implementation implements it as a Map Itself so you can use the map operations and don't have to reinvent things like "get, put, iterator, etc" That can be kind of tricky since collections come with a range of expected behaviors and I can't guarantee that the picad one actually conforms to all the requirements.